### DISCUSSION TITLE
variabal in blockManager

### DISCUSSION THREAD

**Discussion by gitwithsarthak** [GrapesJS/grapesjs#4005] [Q&A]
can we store variables in blockManager block?

blockManager:{
					blocks:[
							{
								id:'sliders-1',
								attributes:{class:'gjs-block-sliders1'},
								category:'Sliders',
								content:'{{compo}}',
							}
						    ]
                                   }
const compo = `<h1>My Component</h1>`;

**Comment by aliibrahim123** [comment]
you must use `new`
